src/theme/default-config.js

```diff
@@ -10,6 +10,7 @@ const defaultTheme = {
 				'brand-border-300': '#93C5FD',
 				'brand-400': '#60A5FA',
 				'brand-500': '#3B82F6',
+				'brand-50': '#EFF6FF',
```

Comment:
The color is already defined. Use this class name `brand-background-50`.

Reply:
fixed!